R/linter_tags.R

```diff
@@ -180,10 +183,11 @@ rd_linters <- function(tag_name) {
 #'
 #' @noRd
 rd_taglist <- function() {
-  linters <- available_linters(exclude_tags = NULL)
+  linters <- available_linters()
```

Comment:
This looks a bit hacky.
Maybe instead overwrite the `tags` column for deprecated linters?

Reply:
I'll take another pass once #1961 is done

Reply:
Good idea. It's still pretty hacky, but at least easier to follow. Ditto I hope for the tests.